build/GitCommitInfo.targets

```diff
@@ -27,7 +27,7 @@
 
     <WriteLinesToFile File="$(GitCommitInfoProps)"
                       Lines="$(GitCommitInfoPropsContent)"
-                      Condition=" '$(ShouldOverwriteThePropsFile)' == 'true' "
+                      Condition=" '$(ShouldOverWriteThePropsFile)' == 'true' "
```

Comment:
I can fix it. But curious, are msbuild properties case-sensitive? It seems like they are not, otherwise this shouldn't have worked before.